lib/linguist/languages.yml

```diff
@@ -41,6 +41,14 @@ AGS Script:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+AMPL:
+  type: programming
+  color: "#00008B"
+  extensions:
+  - .ampl
+  tm_scope: none
```

Comment:
Done in https://github.com/ampl/linguist/commit/51af1bd162eccaddf3f5d224dd4cbbbb3385d28e, thanks!